chains/ethereum/types.go

```diff
@@ -29,6 +30,8 @@ func (r RedeemStatus) String() string {
 		return "Success"
 	case 2:
 		return "Expired"
+	case 3:
```

Comment:
is this one -2 or 3?

Reply:
Can be anything , just added -2 to signify that its different , the rest of status are returned directly from smart contract